data/yam/agama/auto/scripts_ssh_pub_key.jsonnet

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
you can also edit the same file instead of creating new one due to this is only used for remote architectures.